AdminUi/src/AdminUi.Infrastructure.Database.SqlServer/Migrations/20230929144733_ClientsOverview.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace AdminUi.Infrastructure.Database.SqlServer.Migrations
+{
+    /// <inheritdoc />
+    public partial class ClientsOverview : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql("""
+                CREATE OR ALTER VIEW ClientOverviews AS
+                    SELECT
+                        CLIENTS.ClientId,
+                        CLIENTS.DisplayName,
+        	            CLIENTS.DefaultTier,
+                        COUNT (IDENTITIES.ClientId) AS NumberOfIdentities
+                    FROM Devices.OpenIddictApplications CLIENTS
+                    LEFT JOIN Devices.Identities IDENTITIES
+                    ON IDENTITIES.ClientId = CLIENTS.ClientId
+                    GROUP BY CLIENTS.ClientId, CLIENTS.DisplayName, CLIENTS.DefaultTier
```

Comment:
Isn't the following easier/better?

```sql
SELECT
    CLIENTS.ClientId,
    CLIENTS.DisplayName,
    CLIENTS.DefaultTier,
    (
		SELECT COUNT(ClientId) 
		FROM Devices.Identities
		WHERE ClientId = CLIENTS.ClientId
	) AS NumberOfIdentities
FROM Devices.OpenIddictApplications CLIENTS
```
(honest question, because I'm not an SQL expert by any means. ;)

Reply:
Yeah it seems much better, the one I had before was overly complicated. I tested it and it seems like there should be no issues.